### PR TITLE
Create media object 

### PR DESCRIPTION
**Pull request by alseny-diallo**

### CODE REVIEW

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/MediaObject.razor [hunk 1]
```diff
@@ -1,6 +1,146 @@
...
+                    <img src="https://get.foundation/sites/docs/assets/img/media-object/avatar-1.jpg" />
+                </OpenCodeDev.Blazor.Foundation.Documentation.Thumbnail>
+            </MediaObjectSection>
+            <MediaObjectSection>
+                <h4>Dreams feel real while we're in them.</h4>
+                <p>I'm going to improvise. Listen, there's something you should know about me... about inception. An idea is like a virus, resilient, highly contagious. The smallest seed of an idea can grow. It can grow to define or destroy you.</p>
+            </MediaObjectSection>
+        </MediaObjectWrapper>
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.Callout TypeClass="primary">
+            In flexbox mode, the class <Code>.main-section</Code> must be added to your center section in order to properly size it.
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Containers/MediaObject.razor:38]: dont add . before class except for native html tags told ya b4

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/MediaObject.razor [hunk 2]
```diff
@@ -1,6 +1,146 @@
...
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Section Alignement</h2>
+        <p>
+            Each section aligns to the top by default, but individual sections can also be middle- or bottom-aligned with the
+            <Code>.middle</Code> and <Code>.bottom</Code> classes.
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Containers/MediaObject.razor:52]: dont add . before class except for native html tags told ya b4

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/MediaObject.razor [hunk 3]
```diff
@@ -1,6 +1,146 @@
...
+                <p>What is the most resilient parasite? Bacteria? A virus? An intestinal worm? An idea. Resilient... highly contagious. Once an idea has taken hold of the brain it's almost impossible to eradicate. An idea that is fully formed - fully understood - that sticks; right in there somewhere.</p>
+            </MediaObjectSection>
+            <MediaObjectSection Class="bottom">
+                <OpenCodeDev.Blazor.Foundation.Documentation.Media.Thumbnail>
+                    <img src="https://get.foundation/sites/docs/assets/img/media-object/avatar-3.jpg" />
+                </OpenCodeDev.Blazor.Foundation.Documentation.Media.Thumbnail>
+            </MediaObjectSection>
+        </MediaObjectWrapper>
+        <p>
+            In flexbox mode, you can use the >flexbox helper classes instead to get the same result. The <code>.align-*</code> classes
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Containers/MediaObject.razor:74]: dont add . before class except for native html tags told ya b4

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/MediaObject.razor [hunk 4]
```diff
@@ -1,6 +1,146 @@
...
+            can be used on the container to align every child section at once, or individual child sections can be aligned with
+            <code>.align-self-*</code> classes.
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Containers/MediaObject.razor:76]: dont add . before class except for native html tags told ya b4

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/MediaObject.razor [hunk 5]
```diff
@@ -1,6 +1,146 @@
...
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Stack on Small</h2>
+        <p>
+            By adding the <code>.stack-for-small</code> class, you can make your media object responsive. Images will get a width of 100%,
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Containers/MediaObject.razor:89]: dont add . before class except for native html tags told ya b4

**Review by maxgsamson — CHANGES_REQUESTED** [review]
see comments

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/MediaObject.razor (hunk 5 above)
Comment (alseny-diallo) [reply Blazor_Foundation_6_Test_Shared/Documentation/Containers/MediaObject.razor]: i have made the changes

**Review by maxgsamson — APPROVED** [review]
All Passed

**Pull request merged**